docs/ui/components/Home/sections/QuickStart.tsx

```diff
@@ -52,6 +53,50 @@ export function QuickStart() {
             Start Tutorial
           </HomeButton>
         </GridCell>
+        <GridCell
+          className={mergeClasses(
+            'bg-palette-pink3 border-palette-pink7 relative z-0 min-h-[164px] dark:bg-palette-pink3',
+            'max-md-gutters:min-h-[200px]'
+          )}>
+          <RouterLogo
+            className={mergeClasses(
+              'size-20 absolute right-7 bottom-7 border-[5px] rounded-xl p-3',
+              'stroke-[0.01rem] stroke-palette-pink8 text-palette-pink8 bg-palette-pink4 border-palette-pink5'
+            )}
+          />
+          <RawH2 className="!text-palette-pink11 relative z-10 !text-lg">
+            Get what's the best in file-based routing with Expo Router
+          </RawH2>
+          <HomeButton
+            className="bg-palette-pink10 border-palette-pink10 dark:text-palette-pink2 hocus:bg-palette-pink9"
+            href="/router/introduction/"
+            size="sm"
+            rightSlot={<ArrowRightIcon className="icon-md dark:text-palette-pink2" />}>
+            Discover more
```

Comment:
```suggestion
            Learn more
```

After the previous update, this can be changed as well.